pkg/v7/key/key.go

```diff
@@ -54,6 +54,62 @@ func ClusterOrganization(clusterGuestConfig v1alpha1.ClusterGuestConfig) string
 	return clusterGuestConfig.Owner
 }
 
+// CommonChartSpecs returns charts installed for all providers.
+// Note: When adding chart specs you also need to add the chart name to the
+// desired state test in the chartconfig service.
+func CommonChartSpecs() []ChartSpec {
```

Comment:
All the state moves to the key package. This is for charts installed for all providers.